ivy/functional/ivy/experimental/manipulation.py

```diff
@@ -1981,3 +1981,68 @@ def unique_consecutive(
         counts=ivy.array([2, 2, 1, 2, 1]))
     """
     return ivy.current_backend(x).unique_consecutive(x, axis=axis)
+
+
+@handle_exceptions
+@handle_nestable
+@handle_array_like_without_promotion
+@to_native_arrays_and_back
+@handle_array_function
+def trim_zeros(
+    filt: Union[
```

Comment:
Please include native list/tuple typehint as well.

Reply:
Changed typehints to be `Union[ivy.Array, ivy.NativeArray]`